tests/scattering2d/test_jax_scattering2d.py

```diff
@@ -0,0 +1,285 @@
+import os
+import io
+import jax.numpy as jnp
+import numpy as np
+from kymatio import Scattering2D
+from collections import namedtuple
+import pytest
+
+
+backends = []
+
+from kymatio.scattering2d.backend.jax_backend import backend
+backends.append(backend)
+
+
+class TestPad:
+    @pytest.mark.parametrize('backend', backends)
+    def test_Pad(self, backend):
+        pad = backend.Pad((2, 2, 2, 2), (4, 4), pre_pad=False)
+
+        x = np.random.randn(4, 4) + 1J * np.random.randn(4, 4)
+        x = x[np.newaxis, ...]
+
+        z = pad(x)
+
+        assert z.shape == (1, 8, 8)
+        assert z[0, 2, 2] == x[0, 0, 0]
+        assert z[0, 1, 0] == x[0, 1, 2]
+        assert z[0, 1, 1] == x[0, 1, 1]
+        assert z[0, 1, 2] == x[0, 1, 0]
+        assert z[0, 1, 3] == x[0, 1, 1]
+
+        pad = backend.Pad((2, 2, 2, 2), (4, 4), pre_pad=True)
+
+        x = np.random.randn(8, 8) + 1J * np.random.randn(8, 8)
+
+        z = pad(x)
+
+        assert np.allclose(x, z)
+
+    @pytest.mark.parametrize('backend', backends)
+    def test_unpad(self, backend):
+        x = np.random.randn(4, 4) + 1J * np.random.randn(4, 4)
+
+        y = backend.unpad(x)
+
+        assert y.shape == (2, 2)
+        assert y[0, 0] == x[1, 1]
+        assert y[0, 1] == x[1, 2]
+
+
+class TestModulus:
+    @pytest.mark.parametrize('backend', backends)
+    def test_Modulus(self, backend):
+        modulus = backend.modulus
+
+        x = np.random.rand(100, 10, 4) + 1J * np.random.rand(100, 10, 4)
+
+        y = modulus(x)
+        u = np.squeeze(np.sqrt(np.real(x) ** 2 + np.imag(x) ** 2))
```

Comment:
what is the squeeze for?

I would opt for `x.real, x.imag`

Reply:
the squeeze is unnecessary